spring-integration-core/src/main/java/org/springframework/integration/support/leader/LockRegistryLeaderInitiator.java

```diff
@@ -92,6 +87,17 @@
 	 */
 	private final Candidate candidate;
 
+	/**
+	 * flag to denote whether the {@link ExecutorService} was provided via the setter and
```

Comment:
I would say JavaDocs on the properties don't make sense, more over if they don't start from the capital letter 😄

Reply:
Given that the variable was renamed to `executorServiceExplicitlySet`, based on your feedback, it doesn't really convey any meaning as to its use, hence why I previously called it `shutdownExecutorService` 

Without a more meaningful name I felt that putting a JavaDoc on it to explain its use was wise, even though the code should explain what is doing rather than any commentary

I have capitalised the first letter of the docs, if you don't want any docs on this property then I can remove - if you feel the comment is not appropriate then please provide an example of what you feel would be more appropriate